### PR TITLE
[p5.js hi] translate example 06-10

### PR DESCRIPTION
**Pull request by Surbhi-Pittie**

### CODE REVIEW

File: src/content/examples/hi/06_Calculating_Values/00_Interpolate/description.mdx [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-halfway between 0 and 10. Different types of interpolation use
-different rates of change between values. Linear interpolation,
-abbreviated as lerp, uses a constant rate of change. The
-<a href="https://p5js.org/reference/#/p5/lerp" target="_blank">lerp()</a>
-function linearly interpolates between two numbers.
+दो अन्य मानों के बीच मान की गणना करता है। उदाहरण के लिए, संख्या 5 है
+0 और 10 के मध्य। विभिन्न प्रकार के प्रक्षेप उपयोग
+मूल्यों के बीच परिवर्तन की विभिन्न दरें। रेखिक आंतरिक,
+जिसे संक्षेप में एलआरपी कहा जाता है, परिवर्तन की निरंतर दर का उपयोग करता है।
+<a href='https://p5js.org/reference/#/p5/lerp' target='_blank'>lerp()</a>
```
Comment (Divyansh013) [inline src/content/examples/hi/06_Calculating_Values/00_Interpolate/description.mdx:12]: use double quotes only for link

File: src/content/examples/hi/06_Calculating_Values/03_Constrain/description.mdx [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
-keeps the circle within a rectangle. It does so by passing the
-mouse's coordinates into the
-<a href="https://p5js.org/reference/#/p5/constrain" target="_blank">constrain()</a>
-function.
\ No newline at end of file
+यह उदाहरण कर्सर की स्थिति के अनुसार एक वृत्त खींचता है
+वृत्त को एक आयत के भीतर रखता है। यह पारित करके ऐसा करता है
+माउस के निर्देशांक में
+<a href='https://p5js.org/reference/#/p5/constrain' target='_blank'>constrain()</a>
+समारोह।
```
Comment (Divyansh013) [inline src/content/examples/hi/06_Calculating_Values/03_Constrain/description.mdx:11]: use function only

File: src/content/examples/hi/07_Repetition/01_Color_Wheel/description.mdx [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
-<a href="https://p5js.org/reference/#/p5/for" target="_blank">for loop</a>
-to repeat transformations. The loop initializes
-a variable called angle, which changes the rotation of a circle as
-well as its hue. Each time the loop repeats, a circle is drawn
-relative to the center of the canvas. The
-<a href="https://p5js.org/reference/#/p5/push" target="_blank">push()</a>
-and <a href="https://p5js.org/reference/#/p5/pop" target="_blank">pop()</a>
-functions make these transformations affect only the individual circle.
\ No newline at end of file
+यह उदाहरण भिन्न-भिन्न की उपस्थिति को दर्शाता है
```
Comment (Divyansh013) [inline src/content/examples/hi/07_Repetition/01_Color_Wheel/description.mdx:7]: रंग should be used just after भिन्न-भिन्न

**Review by Divyansh013 — CHANGES_REQUESTED** [review]

**Review by Divyansh013 — APPROVED** [review]

**Comment by Divyansh013** [comment]
@TanviKumar PTAL

**Review by TanviKumar — APPROVED** [review]

**Comment by Divyansh013** [comment]
We can merge this PR @limzykenneth @Qianqianye

**Comment by Qianqianye** [comment]
Looks great, thanks!

**Pull request merged**